src/components/header/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import glamorous from 'glamorous';
+
+export const HeaderLeft = ({ children }) => <Fragment>{children}</Fragment>;
+
+HeaderLeft.propTypes = {
+  children: PropTypes.element.isRequired,
```

Comment:
use PropTypes.node and pas requis (l'idée est d'avoir un composant flexible)

Reply:
ok